transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/Message.kt

```diff
@@ -51,6 +51,8 @@ data class Message(
         Text,
         Event,
         QuickReply,
+        Card,
```

Comment:
Seems like `Card` type is never used. Are we going to leave only Carousel option? If yes, we should remove it. Another option is to have Message.Type.Cards (combination of carousel (multiple cards) and single card).